Formula/ldc.rb

```diff
@@ -69,7 +74,7 @@ def install
 
       if OS.mac?
         # Workaround for https://github.com/ldc-developers/ldc/issues/3670
-        cp Formula["llvm"].opt_lib/"libLLVM.dylib", lib/"libLLVM.dylib"
+        cp llvm.opt_lib/"libLLVM.dylib", lib/"libLLVM.dylib"
```

Comment:
I should replace this workaround with either a symlink, or setting `CMAKE_INSTALL_RPATH`. We already have a runtime dependency on LLVM anyway. Will consider doing that if I have to rebase this.